polyfill/spatial-navigation-polyfill.js

```diff
@@ -147,14 +147,18 @@
 
     // 2 Optional step, UA defined starting point
     if (startingPoint) {
-      elementFromPosition = (document.elementFromPoint(startingPoint.x, startingPoint.y)).getSpatialNavigationContainer();
+      // if there is a starting point, set eventTarget as the element from position for getting the spatnav container
+      elementFromPosition = document.elementFromPoint(startingPoint.x, startingPoint.y);
     }
 
-    // 3
-    if (elementFromPosition && searchOrigin.contains(elementFromPosition)) {
-      eventTarget = elementFromPosition;
-    } else {
+    // 3 use starting point
+    // 1) starting point is inside the spatnav container
+    // 2) starting point is inside the non-focusable element
+    if ((searchOrigin === elementFromPosition) && !isContainer(searchOrigin)) {
       eventTarget = searchOrigin;
+      startingPoint = null;
+    } else {
+      eventTarget = elementFromPosition;
```

Comment:
In case there is no startingPoint,  eventTarget could be null.

Reply:
I've fixed this with https://github.com/WICG/spatial-navigation/pull/176/commits/d70e6dd78e8560165cc1583fe0bdb72c7beb49b6.